src/dashboards.js

```diff
@@ -323,4 +323,41 @@ var indicate = (function() {
       $(".busy-indicator").hide();
     }
   }
-})();
\ No newline at end of file
+})();
+
+function compressOSs() {
```

Comment:
i see you decided to keep compressOSs -- I guess that works too. But can you add some kind of indicator that the "Windows"/"Linux"/"Darwin" *labels* are clickable.. like maybe "Windows (click for all)"?

Also if the user has selected by OS (e.g. WINNT), it shouldn't say "7 OSs", it should say "Windows".
Alternately it should say "Windows, OS X" or "Windows, OS X, Linux" if two or three entire OS familes are selected